subsys/net/lib/wifi_provision/proto/common.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+enum Band {
+  BAND_UNSPECIFIED = 0;
```

Comment:
This needs to be renamed to BAND_ANY based on the Wifi provisioner over BLE. In  addition SoftAP adds support to 6_GHZ so may be this needs to be reflected on the BLE example as well?